README.md

```diff
@@ -154,61 +160,29 @@ Also, if you would like docker-compose to run in daemon mode (which means that i
 # Viewing the Application
 ## if you chose the local route, then you are good to go on:
   http://localhost:3000
-  $ rspec (to run the test suite) or bundle exec rspec (if the first does not work)
-  
+  $ `rspec` (to run the test suite), or, `bundle exec rspec` (if the `rspec` does not work)
+
+
 # Other Tips to Get Started
 
-## Loggin In
+## Logging In
 A username and password to log in and test the app are in `seeds.rb`
 
 ## Testing
-When writing tests for rspec tests within the spec/request directory, you can use Warden::Test:Helpers
+* Running `bin/test` will run ruby tests (rspec) and then the js tests (mocha) if all the rspec tests pass
+* To run front end tests and back end tests individually:
+    * Vue.js front end tests: `yarn test` or `yarn test -w` (mocha and chai are included in the `package.json` )
+    * Rails front end and back end tests: `rspec` (rspec is included in the Gemfile)
+
+When writing rspec tests within the spec/request directory, you can use `Warden::Test:Helpers`
 which give you access to the ```login_as(user, :scope => :user)``` method, as well as the ```logout``` method.
-You use FactoryBot.create(:user) before the login_as method and pass it in as the required resource variable.
-BE SURE to include the line ```after { Warden.test_reset! } ``` after the before do block with the login_as method
+You use `FactoryBot.create(:user)` before the `login_as` method and pass it in as the required resource variable.
+BE SURE to include the line ```after { Warden.test_reset! } ``` after the `before do` block with the `login_as` method
 within it. This allows for any unexpected state data of the user from hanging around and causing errors.
 
-Additional testing for front_end specs should make use of Capybara ```sign_in/sign_out``` Capybara methods.
-
-## App Startup Troubleshooting
-If you Receive an error when trying to run
 
-```
-$ heroku local -f Procfile.dev
```

Comment:
should we be trying to keep this heroku local stuff in here? if so, we need to add a Procfile so ppl can get past this